src/components/expandable-content/expandable-container.tsx

```diff
@@ -30,6 +30,7 @@ export const ExpandableContainer: React.FC<IProps> = (props) => {
           <EmbeddablePluginSideTip
             key={sideTip.embeddable.ref_id}
             embeddable={sideTip.embeddable}
+            {...(pluginsLoaded && {pluginsLoaded: pluginsLoaded})}
```

Comment:
perhaps you tried it, but I think you do 
`pluginsLoaded={pluginsLoaded}` here.
or perhaps even
`{..., pluginsLoaded}`

Reply:
I see that in theory you don't need to add the pluginsLoaded property, because on line 24, the sideTips array will be empty until pluginsLoaded is true. But It seem good for consistency to add this reduancy.
I'd just suggest adding a comment something like:
"technically the EmbeddablePluginSideTip won't be rendered if the plugins are not loaded yet, but passing pluginLoaded into it makes it more consistent with the rest of the plugins components"

Reply:
I did try `pluginsLoaded={pluginsLoaded}` first, but it caused an error. I've tried it again though and now it works. I'm not sure what changed, but it seems to be fine.